modFastVM/build.gradle

```diff
@@ -9,6 +9,9 @@ ext.moduleName = 'aion.fastvm'
 
 test.dependsOn copyNativeLibsForModuleTests
 
+targetCompatibility = 10
+sourceCompatibility = 10
```

Comment:
11?

Reply:
this is for the gradle testing issue, must set to the ver10 so far